static/app/views/performance/http/httpLandingPage.tsx

```diff
@@ -108,7 +108,7 @@ export function HTTPLandingPage() {
   const domainsListResponse = useSpanMetrics({
     search: MutableSearch.fromQueryObject(tableFilters),
     fields: [
-      'project.id',
+      'project',
```

Comment:
🤔 I think you need both here, or the links will break, no? The `project.id` is used to construct the URL in the link

Reply:
Link works for me, looks like the `<Projects>` component internally figures out the id's from the slug

https://github.com/getsentry/sentry/blob/5bdbbb6a3b758cfd7ddd139444a822b9ceceefc1/static/app/utils/discover/fieldRenderers.tsx#L308

Reply:
I mean the link to the domain summary page, rather than the link to the project, sorry!

Reply:
Yes you're right! Nice catch! I'll add that back for this PR.

Might be an unnecessary optimization, but I do wonder if we should/could also just pull id from the slug in the DomainCell component. Looking at the projects component, it appears that all the projects are stored in some local state.

Reply:
Added `project.id` back for http and cache to the query (Cache also needs the project.id for a similar reason)

Reply:
Definitely maybe! There are a lot of shenanigans with pulling project slugs and IDs from the backend